_posts/2020-06-23-case-principal-transformer.adoc

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: "Using case principal transformers in Elytron"
+date: 2020-06-23
```

Comment:
This can wait until closer to the WildFly 22 release but we'll want to update the date here and in the `.adoc` file name.